float32[7] msrJntPos
float32[7] cmdJntPos
float32[7] cmdJntPosFriOffset
float32[7] msrJntTrq
float32[7] estExtJntTrq
float32[7] gravity